integration_test/dr-buckets.tf.template

```diff
@@ -0,0 +1,154 @@
+resource "aws_s3_bucket" "orca-archive-worm" {
```

Comment:
I am not seeing where you are creating a policy for the WORM bucket. I would imagine we would need one similar to the primary with some tweaks.

Reply:
added versioning to the bucket according to bucket policy instructions webpage.

Are you also suggesting S3 uploads should limit to GLACIER and DEEP_GLACIER similar to orca-primary bucket?

Reply:
will add policy for s3 uploads and s3 object lock

Reply:
added